docs/source/en/_toctree.yml

```diff
@@ -63,12 +63,10 @@
     title: Pipeline callbacks
   - local: using-diffusers/reusing_seeds
     title: Improve image quality with deterministic generation
-  - local: using-diffusers/control_brightness
-    title: Control image brightness
+  - local: using-diffusers/pipeline_techniques
```

Comment:
We're only focusing on image enhancement in the corresponding guide. So, in that light, `pipeline_techniques` seems to be a little too general.